src/views/DocumentationTopic.vue

```diff
@@ -289,6 +290,9 @@ export default {
         combineVersions(topicDataDefault.schemaVersion), MIN_RENDER_JSON_VERSION_WITH_INDEX,
       ) >= 0
     ),
+    enableOnThisPageNav: ({ isTargetIDE, store }) => (
+      !isTargetIDE && store.state.onThisPageSections.length > 2
```

Comment:
I think we might want to consider `> 1` instead of `> 2`. Browsing the documentation for DocC, I was confused at first when the nav didn't show up, because there were only two sections (Overview and Topics), but it seemed like it would have been useful there.

Reply:
There are two debatable options here:
1. Always show the OTP (so it doesn't look broken when not present; assume it is always useful info)
2. Remove the OTP when there are >2 items; assume those items are not helpful and it is repetitive info

I can see arguments for both. @mportiz08 and @ethan-kusters I think you two have differing opinions.

Reply:
My main objection here is that it's easy to end up with 2 autogenerated sections (via the declaration and topics section for example) on a page that has little to no content: https://apple.github.io/swift-argument-parser/documentation/argumentparser/argumentvisibility.

We could resolve that by looking at page height or but ignoring autogenerated content but those are both problematic in their own way.

Essentially I'd like to avoid including an "On this Page" for a page that doesn't use the scroll bar to make all the headings visible. If I can already see what's on this page by looking at the page (without scrolling), the extra navigation element is (to me) a little confusing.

Reply:
🤔 we could easily check whether there is a scrollbar or not...

Reply:
It still provides a highlight of the subsections that doesn't require the user manually reading and parsing through what they are—at a glance, they can see what's on the page without the actual paragraph text getting in the way. I wouldn't necessarily say it's a huge benefit or anything, but I don't think it's very distracting on short pages, especially since you have to already have a very wide screen just to see the affordance. For me personally, it would be much more confusing when it sometimes disappears without an obvious reason why. That's just my personal opinion though.

Reply:
And the scenario I originally pointed out could happen even on a page with lots of non-auto-generated content (like the DocC example). For example, if you have a large overview and a topics section, like this one: https://www.swift.org/documentation/docc/